### DISCUSSION TITLE
Redirect within a script

### DISCUSSION THREAD

**Discussion by ftjfo** [fish-shell/fish-shell#9682] [General]
In bash you could do something like this in order to redirect all output within a script to a file.  Any equivalent approach in fish?

```
#!/bin/bash
exec &> >(tee -a /path/to/logfile)
```

**Comment by faho** [comment]
Fish does not currently have global redirections like that, no. This is #3948.

**Reply by ftjfo** [reply]
Thanks, I did see that previous issue, but it was 6 years ago so perhaps new features were added.
